src/inversion_scripts/calc_sensi.py

```diff
@@ -106,7 +129,12 @@ def process(h):
                 # Get the data for the current hour
                 pert = pert_data["SpeciesConcVV_CH4"][h, :, :, :]
                 # Compute and store the sensitivities
-                sensitivities = (pert.values - base.values) / perturbation
+                if (perturbationBC is not None) and (e >= (nelements-4)):
+                    sensitivities = (pert.values - base.values) / perturbationBC
+                    if h != 0: # because we take the first hour on the first day from spinup
```

Comment:
I'm probably missing something, but what about when `h == 0` on other days (besides the first one)? And why do we need to perform the `test_GC_output_for_BC_perturbations` check for every time step?

Reply:
This just ensures that the BC perturbation in GEOS-Chem is behaving correctly to make sure, for example, someone isn't using an incompatible version of GEOS-Chem. @nicholasbalasus put this check in so I will defer to him, but.... My suspicion is that we don't necessarily need to do the check for every time step, and so skipping `h ==0` shouldn't matter for other days, as long as the other hours are correct. It's a pretty simple check so computationally shouldn't matter, but I suppose we could just do a spot check for each day eg. `if h == 1`

Reply:
Either should be fine. The more checks the better (it should, but in case the perturbation doesn't work during some hours), but performance should be considered as well.